File: assignment1/models/ManufacturedItem.java
package assignment1.models;

public class ManufacturedItem extends Item {
	private double tax;
	public ManufacturedItem(double price) 
	{
		super(price);		
		tax = 12.5 / 100 * price; 
        tax += 2.0 / 100 * (price + tax);		
	}
	public Double fetchingTax() 
	{
		 // instead of return the whole object, returned a variable of it
	     return tax;
	}

 
}

File: dart_assignment1/lib/component/manufactured_item.dart
import 'package:dart_assignment1/component/item_factory.dart';
import 'item.dart';

class ManufacturedItem extends Item {
  ManufacturedItem(String name, double price, int quantity, ItemType type)
      : super(name, price, quantity, type) {
    tax = 12.5 / 100 * price;
    tax += 2.0 / 100 * (price + tax);
  }
  //noSuchMethod(Invocation invocation) => super.noSuchMethod(invocation);
}
